cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CurrentLimitsMapping.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.conversion;
+
+import com.powsybl.iidm.network.CurrentLimitsAdder;
+import com.powsybl.iidm.network.Terminal;
+
+import java.util.*;
+import java.util.function.Function;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public class CurrentLimitsMapping {
+
+    private final Map<Terminal, CurrentLimitsAdder> adders;
```

Comment:
This map will grow during the conversion: a terminal is never removed, even the limits have been created. Should it really be in thee context: are date spread in the file? Could it be a local variable or less global variable?

Reply:
This is mandatory as nothing prevents to write several `OperationalLimitSet` for the same terminal and/or equipment.